src/HealthChecks.Consul/ConsulOptions.cs

```diff
@@ -7,5 +7,9 @@ public class ConsulOptions
         public int Port { get; set; }
 
         public bool RequireHttps { get; set; }
+

```

Comment:
Could be a good idea to add RequireBasicAuthentication as boolean and use it to add the header in the request?